README.md

```diff
@@ -174,6 +174,53 @@ kubectl api-resources
 
 Check out our [Contributing](CONTRIBUTING.md) and [Developer](docs/developers) guides.
 
+## Using Workspaces
+
+The workspaces are arranged in a tree, like directories in a
+filesystem.  The root is known as `root`.  Pathnames use colon (`:`)
+as the separator, rather than the forward or backward slash commonly
+found in filesystems.  An absolute pathname is one that starts with
+`root`.  For example: when I started kcp and went to my home
+directory, I found its absolute pathname to be
+`root:users:zu:yc:kcp-admin`.
+
+There are a few different types of workspace built into the kcp
+binary, and developers can add more workspace types by creating API
+objects of type `ClusterWorkspaceType` in the root workspace.  You can
+see the list of available types by doing `kubectl get
+ClusterWorkspaceType` in the root workspace.
+
+Only certain pairings of parent and child workspace type are allowed.
+Each type of workspace may limit the types of parents and/or the
+types of children that are allowed.
+
+Following is a description of the path to my home workspace.
```

Comment:
```suggestion
Here is an example set of workspaces:
```

Reply:
It is not just a set of examples.  It is one _path_ , intended to show the interactions among workspace types and position in the path.

Reply:
We should avoid “my” in this kind of doc. Also, it needs clarification: I couldn't work out what a _path_ is here.